src/main/java/lenra/components/ActionableSchema.java

```diff
@@ -0,0 +1,358 @@
+
+package lenra.components;
```

Comment:
The base package should be `io.lenra`: 

```suggestion
package io.lenra.components;
```